platform/src/test/java/org/exoplatform/platform/qa/ui/platform/gatein/GateinGlobalSettingsTestIT.java

```diff
@@ -0,0 +1,257 @@
+package org.exoplatform.platform.qa.ui.platform.gatein;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAddManagement;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAndGroupManagement;
+import org.exoplatform.platform.qa.ui.selenium.logger.Logger;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+
+
+import org.exoplatform.platform.qa.ui.selenium.platform.administration.ChangeLanguages;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.UserProfilePage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import sun.security.krb5.Config;
+
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.core.PLFData.*;
+import static org.exoplatform.platform.qa.ui.selenium.DownloadFileControl.driver;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.ELEMENT_HOME_LINK_PLF;
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.ELEMENT_HOME_LINK_PLF_IN_FRENCH;
+import static org.exoplatform.platform.qa.ui.selenium.locator.social.SocialLocator.ELEMENT_NAME_OF_USER_TOP_RIGHT;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+
+/**
+ * @date 8 April 2015
+ * @author tult
```

Comment:
To remove.
Instead please add a real description of the class.

Reply:
done